views/config_notifications.mako

```diff
@@ -1604,6 +1608,7 @@
                                             <input type="checkbox" class="enabler" name="trakt_sync_watchlist" id="trakt_sync_watchlist" ${'checked="checked"' if app.TRAKT_SYNC_WATCHLIST else ''}/>
                                             <p>sync your Medusa show watchlist with your trakt show watchlist (either Show and Episode).</p>
                                             <p>Episode will be added on watch list when wanted or snatched and will be removed when downloaded </p>
+                                            <p><b>Note:</b> By design, Trakt auto remove episode and/or show from watchlist as soon you watched it.</p>
```

Comment:
By design, Trakt automatically removes episodes and/or shows from watchlist as soon you have watched them.